### DISCUSSION TITLE
Add annotation in produced HTML for the language used

### DISCUSSION THREAD

**Discussion by colelawrence** [code-hike/codehike#238] [Ideas and feedback]
I am building a docs site with contentful and I would like to be able to see what language was actually used to produce the React `.ch-code-scroll-parent` element. Something like a `data-ch-language="ts"` would be extraordinary for my use-case, where I need to be able to read code blocks off of my own page coming out of the MDX content, so I can insert their source code contents into a demo iframe.

Thanks!

**Comment by pomber** [comment]
Sure, seems like a good attribute to have. I'll try to add it in the next version.

**Reply by pomber** [reply]
Released in [v0.7.3-next.3](https://github.com/code-hike/codehike/releases/tag/v0.7.3-next.3)
